Added Custom Services Section in readme (#42)
commercetools/commercetools-dotnet-core-sdk: README.md

```c#
var command = ((ClientQueryProvider<Category>) query.Provider).Command;
```

> ## Custom Services

Reviewer: Question: why does Stores need to be handled differently than other endpoints?

Author: it's just for example as it's not currently supported, it will be handled soon like other endpoints

> If you want to support services endpoints that still not covered by the SDK like [Stores](https://docs.commercetools.com/http-api-projects-stores#stores ) endpoint for example, all what you need is to create the domain models like described

Reviewer: ```suggestion
To support services endpoints the SDK does not currently support, like the [Stores](https://docs.commercetools.com/http-api-projects-stores#stores ) endpoint, to create the domain models as follows: 

```

```c#
    [Endpoint("stores")]
    public class Store : Resource<Store>
    {
        public string Key { get; set; }
        public LocalizedString Name { get; set; }
    }
    
    public class StoreDraft : IDraft<Store>
    {
        public string Key { get; set; }
        public LocalizedString Name { get; set; }
    }
```

> Please notice that, you have to add the Endpoint attribute above the main domain model, make it inherit from Resource class, and make the draft model implement IDraft interface.

Reviewer: ```suggestion
Note that you have to add the Endpoint attribute **above** the main domain model. The Endpoint attribute then must inherit from the `Resource` class, and the draft model should implement the `IDraft` interface.
```

> After this you can use them in different commands like this:

Reviewer: ```suggestion
After this you can use the custom services in different commands as follows:
```

```c#
 var storeDraft = new StoreDraft
            {
                Name = new LocalizedString {{"en", $"Store1"}},
                Key = $"StoreKey"
            };
 var store = this.client.ExecuteAsync(new CreateCommand<Store>(storeDraft)).Result;
```

or like this:
 ```c#
  var store = this.client..ExecuteAsync(new GetByIdCommand<Store>(id)).Result;
 ```

>  you can also create custom Commands, HttpApiCommands, Request Builders and Additional Parameters builders for them, but don't forget to inject them into services container before calling services.UseCommercetools

Reviewer: ```suggestion
 You can also create custom Commands, HttpApiCommands, Request Builders and Additional Parameters builders for them, but don't forget to inject them into services container before calling `services.UseCommercetools()`.
```
